addon/components/new-osf-navbar/template.hbs

```diff
@@ -0,0 +1,75 @@
+<div class="new-osf-navbar-styling">
+    <nav class="navbar navbar-inverse" id="navbarScope">
+        <div class="container">
+            <div class="navbar-header">
+                {{!Toggle navigation button for xs screens}}
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".secondary-navigation" {{action 'closeSearch'}}>
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
```

Comment:
To a user who can see, this will show an icon with three lines. But to a blind user with a screen reader, there is no indication what this element does. Might I suggest adding aria label text? Here is a general reference for another icon library that may serve as inspiration:
http://fontawesome.io/accessibility/